Apps/AdminWebClient/src/Server/Controllers/CsvExportController.cs

```diff
@@ -94,20 +90,22 @@ public HttpResponseMessage GetComments(DateTime? startDate = null, DateTime? end
         /// <response code="401">the client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpGet]
+        [Route("GetNotes")]
         [Produces("text/csv")]
-        public HttpResponseMessage GetNotes(DateTime? startDate = null, DateTime? endDate = null)
+        public IActionResult GetNotes(DateTime? startDate = null, DateTime? endDate = null)
         {
             return SendContentResponse("Notes", this.dataExportService.GetNotes(startDate, endDate));
         }
 
-        private static HttpResponseMessage SendContentResponse(string name, Stream csvStream)
+        private static FileStreamResult SendContentResponse(string name, Stream csvStream)
```

Comment:
I'm doubting this will work for returning a large file. Do we have a way to manually testing this with a dummy large data? or we can approve this PR then in case we have issue with large file later, we can check the following approach that people are using to stream video:
public static async Task<FileStreamResult> SendContentResponse...
   var stream = await OpenFileMemoryStream...;
    return new FileStreamResult(stream, "text/csv");